src/app/article/article.component.scss

```diff
@@ -140,20 +147,24 @@
 
 /* For tablets: */
 @media only screen and (min-width: 600px) and (max-width: 768px) {
-	#single-article-container {
+	:host /deep/ #single-article-container {
```

Comment:
I just needed the ":host /deep/" for blockquote tags. Without having ":host /deep/" in front of #single-article-container, CSS doesn't apply the blockquote tags if they are within the net of #single-article-container. But I solved it by extracting the blockquote class outside of #single-article-container as seen in c909921d0b7ce724f2e62dcdddddffdc06f7a315.